app/javascript/packages/document-capture/context/selfie-capture-enabled.tsx

```diff
@@ -0,0 +1,16 @@
+import { createContext } from 'react';
+
+interface SelfieCaptureEnabledProps {
+  /**
+   * Specify whether to show the selfie capture on the doc capture screen.
+   */
+  selfieCaptureEnabled: boolean;
+}
+
+const SelfieCaptureEnabledContext = createContext<SelfieCaptureEnabledProps>({
+  selfieCaptureEnabled: false,
+});
+
+SelfieCaptureEnabledContext.displayName = 'SelfieCaptureEnabledContext';
+
+export default SelfieCaptureEnabledContext;
```

Comment:
Maybe we should consider another name for the context, like SelfieCapturingContext etc,  the "Enabled" in the name seems unnecessary.

Reply:
I think you're right! Will push up a change shortly.

Reply:
Updated in [497763d](https://github.com/18F/identity-idp/pull/10183/commits/497763d804d68c4f52ae04df1e9a1b7133fc86c6).